tests/jobs/test_jobify.py

```diff
@@ -0,0 +1,35 @@
+from unittest.mock import patch, Mock
+
+from invisible_flow.jobs.jobify import jobify
+
+
+def function_to_wrap(a):
+    return a + 10
+
+
+# @patch('invisible_flow.jobs.jobify.db')
```

Comment:
I would remove this comment too, if it's not saved for a particular reason

Reply:
Removed